src/css/organisms/filter-form.scss

```diff
@@ -56,6 +56,28 @@
       padding-bottom: 0px;
     }
   }
+
+  /* - Internet Explorer 10–11
+   - Internet Explorer Mobile 10-11 */
+  :-ms-input-placeholder {
+    font-size: 0.8rem;
+    padding: 4px;
+  }
+
+  .filter-form-search {
+    padding: $spacing-xs;
+
+    @media #{$mq-md} {
+      width: 100%;
+    }
+    width: 97%;
+    @supports (-webkit-touch-callout: none) {
+      // fix for mobile safari only
+      width: 92%;
```

Comment:
Yeah... that was a fun one work out.

However, there is a full overhaul of the buttons due to land when we merge Mozilla Protocol V12 (which is PRed, but am leaving it till after hard launch), so this might go away then.